Tests/demistomock/demistomock.py

```diff
@@ -859,7 +859,15 @@ def createIncidents(incidents, lastRun=None, userID=None):
     Creates incident in long running execution
 
     Args:
-      incidents (list): List of incident objects to create
+      incidents (list):
+        List of incident objects to create, with the following required keys, and some common optional keys
+            - name (required) - str
+            - type (required) - if not provided will create unclassified incident) - str
+            - labels (required) - list of {"type": _, "value": _} objects
```

Comment:
are you sure labels are required?

Reply:
You're right, they are not required. fixing via suggestion:
```suggestion
            - labels (optonal) - list of {"type": _, "value": _} objects
```